en/docs/reference/configurations/configuring-micro-gw-conf-for-docker.md

```diff
@@ -0,0 +1,62 @@
+# Configuring micro-gw.conf for Docker
+
+You can use environment variables when running Docker to override the configurations in the `<MGW-RUNTIME-HOME>/conf/micro-gw.conf` file.
+
+- The `-e` flag in the docker run command is used to create environment variables. 
+- You need to create the variable name by appending the `micro-gw.conf` configuration hierarchy by using an `_` underscore to define each level of the hierarchy.
+
+## Example 1 - Defining the Listener HTTPS port
+
+Let’s assume that you are going to fetch an environment variable for `httpsPort` under the `[listenerConfig]` configuration in the `micro-gw.conf` file. The following code snippet shows part of the configuration in the `micro-gw.conf` file.
+
+```
+[listenerConfig]
+  host = "0.0.0.0"
+  httpPort = 9090
+  httpsPort = 9095
+  ...
+  ...
+...
+...
+```
+
+As you can see in the above sample, the `httpsPort` is set as `9095`. In order to change value of the `httpsPort` value you do not need to make any changes to the `mico-gw.conf` file. Instead, when you run the Docker command, you need to append the environment variable in the `run` command as shown below.
+
+``` tab="Format"
+docker run -e <environment-variable>='<value-for-environment-variable>' -d <api>:<api-version>
```

Comment:
Is there a way for a user to define multiple environment variables in the "docker run" command?

Reply:
you can, in that case there should be another key-value mapping with `-e` tag.
https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file

Reply:
Updated in https://github.com/wso2/docs-mg/pull/168/commits/81bf3c0f68c7d5696ceadf96d8c15d39cab56368